tests/unit/amo/components/TestRatingManager.js

```diff
@@ -101,9 +101,8 @@ describe(__filename, () => {
       loadSavedReview,
     });
 
-    expect(loadSavedReview.called).toEqual(true);
-    const args = loadSavedReview.firstCall.args[0];
-    expect(args).toEqual({
+    sinon.assert.called(loadSavedReview);
```

Comment:
Whenever you use `assert.calledWith(theFunction, ...)`, it's not necessary to precede it with `assert.called(theFunction)`. In fact, it's worse because the first test failure you would see is not as informative.